test/users/users_create_test.exs

```diff
@@ -268,4 +268,50 @@ defmodule Okta.UsersCreateTest do
                provider_nametype
              )
   end
+
+  test "update_user/4 requests correctly", %{
+    client: client,
+    base_url: base_url,
+  } do
+    user_id = "123"
+    data = %{profile: %{login: "newlogin@acmec.com"}}
+
+    Okta.Tesla.Mock
+    |> Mox.expect(:call, fn
+      request, _opts ->
+        {:ok, expected_body} = Jason.encode(data)
+
+        assert expected_body == request.body
+        assert :put == request.method
+        assert [strict: false] = request.query
```

Comment:
this test `assert [strict: false] = request.query` fails as the API call with `Okta.Users.update_user(client, user_id, data, [strict: true])` sets the query param `strict` to `true`